src/components/search/SearchResult.jsx

```diff
@@ -38,7 +39,8 @@ export const SearchResult = ({
         <p className="body-text">{bodyText}</p>
 
         <div className="metadata">
-          <div>{`${userName}/${repoName}`}</div>
+          <div>{`${userName}/${repoName}`} </div>
+          <div className="comments">{`Total comments ${comments}`}</div>
```

Comment:
Could we do - 

`Comments: ${comments}`

Reply:
Sure, thanks for the advice.